CommonControls/FileTypes/RigidModel/MaterialHeaders/WeightedMaterial.cs

```diff
@@ -210,7 +210,7 @@ public IMaterial CreateEmpty(ModelMaterialEnum materialId, RmvVersionEnum rmvTyp
         {
             var material = new WeightedMaterial()
             {
-                MaterialId = ModelMaterialEnum.weighted,
+                MaterialId = materialId, 
```

Comment:
Why is this changed? 
Changing this could potential lead to corrupt models as other parts of the code might not expect any value in this case